legal-api/src/legal_api/reports/business_document.py

```diff
@@ -541,21 +555,16 @@ def _set_continuation_in_details(self, business: dict):
             jurisdiction_info = {
                     'id': jurisdiction.id,
                     'jurisdiction': location_jurisdiction,
-                    'identifier': jurisdiction.identifier,
-                    'legal_name': jurisdiction.legal_name,
+                    'identifier': jurisdiction.identifier or 'Not Available',
+                    'legal_name': jurisdiction.legal_name or 'Not Available',
                     'tax_id': jurisdiction.tax_id,
                     'incorporation_date': formatted_incorporation_date,
                     'expro_identifier': jurisdiction.expro_identifier,
-                    'expro_legal_name': jurisdiction.expro_legal_name,
+                    'expro_legal_name': jurisdiction.expro_legal_name or 'Not Available',
```

Comment:
Use `Not Available` as default values.

This update is to make sure we display XPRO info in summary only when `expro_identifier` is available instead of displaying XPRO info all the time ([report template reference](https://github.com/bcgov/lear/blob/main/legal-api/report-templates/template-parts/business-summary/foreignJurisdiction.html#L37)). However, we never get XPRO businesses in the extract, and `expro_legal_name` will always be unavailable.

Here is an example.
<img width="509" alt="image" src="https://github.com/user-attachments/assets/8bcf0d01-71df-4d77-a228-38c77bc59d29" />